client/document/index.jsx

```diff
@@ -133,7 +137,11 @@ class Document extends React.Component {
 							>
 								<div className="masterbar" />
 								<div className="layout__content">
-									<WordPressLogo size={ 72 } className="wpcom-site__logo" />
+									{ isJetpackCloudFlow ? (
+										<JetpackLogo size={ 72 } className="wpcom-site__logo" />
```

Comment:
Should we use a different class name for the Jetpack Logo here? The `wpcom-site__logo` no longer makes sense with the Jetpack logo.

Reply:
It is intentional. I left the `wpcom-site__logo` class here since this element still [is contained within `wpcom-site` wrapper](https://github.com/Automattic/wp-calypso/pull/39172/files#diff-3fb371c23faa311830bf33b7e7e855a3R129) and logically belongs to it.

I wouldn't read it as a "WPCOM logo" but rather a "logo within WPCOM site (admin)". I know it is still quite confusing, but changing `wpcom-site` to something like `jetpack-cloud-site` just for Jetpack Cloud because of the logo would be an overkill in my opinion 🙂But I'm happy to hear what's your take on this.

Alternative solution is to break any relation between the `logo` and `wpcom-site` element for Jetpack Cloud and have a completely independent CSS element here like `jetpack-cloud-loader-logo`.

Reply:
I'm happy with that rationale, @delawski. It's confusing without the explanation, but I'm okay to keep it as-is as a simple and straightforward solution.